protocol-tests/src/main/java/fixtures/dpgcustomization/implementation/DpgClientImpl.java

```diff
@@ -663,7 +663,8 @@ public SyncPoller<BinaryData, BinaryData> beginLro(String mode, RequestOptions r
      * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
-     * @return the response body along with {@link PagedResponse} on successful completion of {@link Mono}.
+     * @return pages that you will either return to users in pages of raw bodies, or pages of models following growup
```

Comment:
Unsure why this changed. My PR on typespec didn't touch the response.

Maybe from this https://github.com/microsoft/typespec/pull/7873